integration_tests/lpython_decorator_01.py

```diff
@@ -1,7 +1,7 @@
 from numpy import array
 from lpython import i32, f64, lpython
 
-@lpython
+@lpython(optimisation_flags=["-ffast-math", "-funroll-loops", "-O3"])
```

Comment:
LPython does not support any of these flags.

Consequently, these are flags to be used with the C compiler that compiles the C code generated by our C backend.

Once we support LLVM, the flags we will support are different (we currently only support `--fast`).

So let's figure out how to do this. One option is:
```
@lpython(backend="c", backend_optimisation_flags=["-ffast-math", "-funroll-loops", "-O3"])
@lpython(backend="llvm", backend_optimisation_flags=["--fast"])
```
That might work.

Reply:
Makes sense. Right now the default backend is `c`, so `backend_optimisation_flags=["-ffast-math", "-funroll-loops", "-O3"]` makes sense in this example. Or do you want to make `backend` a compulsory argument (right now its optional and defaults to `c`?

Reply:
Let's make it compulsory if you use `backend_optimisation_flags`, otherwise things can break in the future once we make LLVM the default. Otherwise let's allow `@lpython` as is, and choose a default backend, currently C, but I think our default should actually be LLVM later on.